projects/libs/core-library-typescript/test/block-storage/block-storage.spec.ts

```diff
@@ -117,4 +121,33 @@ describe("Block storage", () => {
       await expect(blockStorage.getBlocks(lastBlock.header.height)).to.eventually.be.eql([exampleBlock]);
     });
   });
+
+  describe("run checkStartupStatus", () => {
+    it("checkStartupStatus should succeed", async () => {
+      const startupStatus = await blockStorage.startupCheck();
+      return expect(startupStatus).to.deep.equal(<StartupStatus>{ name: "block", status: STARTUP_STATUS.OK });
```

Comment:
Please remove these casts from the codebase as a whole.

Reply:
Removed all except where is causes errors such as startup-check-runner.ts